code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -1098,13 +1098,6 @@ mob/living/carbon/human/proc/change_monitor()
 		var/turf/T = get_turf(L)
 		if(!T || L == src || L.stat == DEAD || L.isSynthetic() || L.is_diona() || isvaurca(L) || L.invisibility == INVISIBILITY_LEVEL_TWO)
 			continue
-		var/image/ping_image = image(icon = 'icons/effects/effects.dmi', icon_state = "sonar_ping", loc = src)
```

Comment:
undocumented change, and not a good one imo. using only the descriptors with the ping is difficult, making this ability useless

Reply:
Unintentional, there is an old version of psi ping in the human_powers dm.